### ISSUE TITLE
Missing lang code

### ISSUE THREAD

**Issue by Seirdy** [esparta/esparta.github.io#3]
The `<html>` element is currently missing a language code. [I filed a PR upstream](https://github.com/Xzya/simple-hugo-theme/pull/9), but just thought you should know. A lang code makes screen-readers and machine-translation work more reliably.
